samples/JavaScript/entry.mjs

```diff
@@ -0,0 +1,6 @@
+import bar from './module.mjs';
```

Comment:
@pchaigno - you don't actually want the bayesian classifier to auto-detect the difference between es5, pre-standard es6, and es6

this is expected to change within the next year, and the difference is a contextual usage knowledge thing, rather than a characteristic of the contained source.  the extension effectively reflects whether the surrounding environment is 1) old, 2) from the future, or 3) has transpilers set up.  your classifier explicitly does not consider these issues